docs/gl_objects/deploy_keys.py

```diff
@@ -34,3 +34,15 @@
 # or
 key.delete()
 # end delete
+
+# enable
+key = gl.project_keys.enable(key_id, project_id=1)
```

Comment:
Since the new methods are defined for the ProjectKey class you should only have one example:

````` python
key.enable()
`````